server/filters.go

```diff
@@ -0,0 +1,154 @@
+package server
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/Masterminds/semver"
+	pbVersion "github.com/Percona-Lab/percona-version-service/versionpb"
```

Comment:
is it generated? just trying to understand why it is not included

Reply:
yes